package.json

```diff
@@ -37,7 +35,22 @@
       "import": "./dist/index.mjs",
       "require": "./dist/index.cjs"
     },
-    "./dist/": "./dist/"
+    "./components/*": {
+      "import": "./dist/components/*.mjs",
+      "require": "./dist/components/*.cjs"
+    },
+    "./directives/*": {
+      "import": "./dist/directives/*.mjs",
+      "require": "./dist/directives/*.cjs"
+    },
+    "./functions/*": {
+      "import": "./dist/functions/*.mjs",
+      "require": "./dist/functions/*.cjs"
+    },
+    "./mixins/*": {
+      "import": "./dist/mixins/*.mjs",
+      "require": "./dist/mixins/*.cjs"
+    }
```

Comment:
That will generate lot of required changes on all apps.
Do we actually need to change the dist folder? Or is it only a visual preference?